app/models/project_group.rb

```diff
@@ -103,6 +108,9 @@ def filtered_children
     end
   end
 
+  # For use in specs
+  alias_method :c, :filtered_children
+
```

Comment:
Would it be better to do this in a monkey patch in the spec file?

Reply:
Not a bad idea.